configs/components/facter.rb

```diff
@@ -217,7 +215,7 @@
   end
 
   # Disable tests for platforms that use the default OS toolchain
-  unless platform.name =~ /sles-15|el-8|debian-10/ || platform.is_fedora?
+  if platform.name =~ /debian-[89]|el-[567]|redhatfips-7|sles-(:?11|12)|ubuntu-(:?14.04|16.04|18.04)/ || !platform.is_linux?
     pkg.check do
       tests
     end
```

Comment:
Not related, but we might want to investigate why tests are disabled here.

Reply:
I will run an ad-hoc with enabled test, to see what breaks where